### PR TITLE
[ticket/50]  Add documentation for new files classes

### LINKED ISSUE #50: Add documentation for new files classes in 3.2

### PR DESCRIPTION
**Pull request by marc1706**
Fixes #50

### CODE REVIEW

**Comment by Nicofuma** [comment]
nice

**Comment by marc1706** [comment]
Will try to read through it again. I think the yaml coding style needs some single quotes.

**Comment by VSEphpbb** [comment]
^ Yes please

**Comment by marc1706** [comment]
Updated

File: development/files/upload.rst [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+As of phpBB 3.2, this is changed to:
+
+.. code-block:: php
+
+    $upload = $phpbb_container->get('files.upload');
+    $upload->set_disallowed_content(array());
+    $extensions = $cache->obtain_attach_extensions((($is_message) ? false : (int) $forum_id));
+    $upload->set_allowed_extensions(array_keys($extensions['_allowed_']));
+    $file = ($local) ? $upload->handle_upload('files.types.local', '$local_storage, $local_filedata) : $upload->handle_upload('files.types.form', $form_name);
```
Comment (lavigor) [inline development/files/upload.rst]: Extra quote here?

**Comment by marc1706** [comment]
Removed extra quote

**Pull request merged**